server/service/src/processors/transfer/shipment/delete_inbound_shipment.rs

```diff
@@ -69,6 +69,18 @@ impl ShipmentTransferProcessor for DeleteInboundShipmentProcessor {
         // 6.
         InvoiceRowRepository::new(connection).delete(deleted_inbound_shipment_id)?;
 
+        let store_id = match linked_shipment {
```

Comment:
to make it slightly cleaner, i would do this:
(above)
`let deleted_inbound_shipment_id = &inbound_shipment.invoice_row.id`
`let deleted_inboud_shipment = inbound_shipment.invoice_row`

`InvoiceRowRepository::new(connection).delete(&deleted_inbound_shipment.id)?;`

 then use deleted_inbound_shipment for store_id etc..

Reply:
Would the store_id not belong to the outbound shipment though since it'll be deleted at picked? 🤔